src/routes/DetailsPage.jsx

```diff
@@ -1,13 +1,23 @@
 import { useSelector } from 'react-redux';
+import { useEffect } from 'react';
 import Doctor from '../components/Doctor';
 import { mainPageDoctors } from '../redux/mainPage/mainPageSlice';
 
 const DetailsPage = () => {
   const { clickedDoctor } = useSelector(mainPageDoctors);
+
+  useEffect(() => {
+    if (Object.keys(clickedDoctor).length > 0) {
+      localStorage.setItem('clickedDoctor', JSON.stringify(clickedDoctor));
```

Comment:
We don't need to set clickedDoctor to local storage since we're using redux persist for that. (ref my PR)